.github/workflows/installer.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       # We cannot use v4, as it requires a newer glibc version than some of the
       # tested images provide. See
       # https://github.com/actions/checkout/issues/1487
```

Comment:
A truly unfortunate reason for not being able to bump this to 4.x

Reply:
😢